ArmPkg/Drivers/CpuDxe/PageTableMemoryAllocation.c

```diff
@@ -104,7 +104,7 @@ FindPoolToAllocateFrom (
   @retval A pointer to the allocated page table memory or NULL if the
           allocation failed.
 **/
-STATIC
+static

```

Comment:
The style guidelines have been updated to use `static` over `STATIC` (though don't disallow it), but I don't think edk2 PRs have actually switched to this. I have to go back and see what the GoogleTest decision was here, but I don't quite recall or really am too worried about it if we want to go this direction.

Reply:
https://edk2.groups.io/g/devel/topic/110977230#msg121076

Reply:
That’s what I’m saying that the style guidelines have been updated but I don’t think edk2 has been following that (yet). Certainly no codebase wide PR has happened for it. And changing to lowercase static will break GoogleTest (which isn’t being used here) and I remember talking on the mailing list about some schemes to fix it, but I don’t recall anything being implemented. But maybe I did. More just mentioning that I will probably still following STATIC until edk2 tells us otherwise in PR review, but I don’t care too much, just noting it might cause an issue if we were ever to write a GoogleTest for this, which we probably won’t.